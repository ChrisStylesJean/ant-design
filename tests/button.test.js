import React from 'react';
import renderer from 'react-test-renderer';
import Button from '../components/button/button';

describe('Button', function() {
  it('renders correctly', () => {
    const tree = renderer.create(
      <Button>Follow</Button>
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('renders Chinese characters correctly', () => {
    const tree = renderer.create(
      <Button>按钮</Button>
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
