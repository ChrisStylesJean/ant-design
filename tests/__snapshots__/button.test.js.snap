exports[`Button renders Chinese characters correctly 1`] = `
<button
  className="ant-btn"
  onClick={[Function]}
  onMouseUp={[Function]}
  type="button">
  <span>
    按 钮
  </span>
</button>
`;

exports[`Button renders correctly 1`] = `
<button
  className="ant-btn"
  onClick={[Function]}
  onMouseUp={[Function]}
  type="button">
  <span>
    Follow
  </span>
</button>
`;
